README.md

```diff
@@ -0,0 +1,21 @@
+# Finbot
+
+## Description
+
+Finbot is a chatbot-based platform that teaches personal finance topics. 
+
+We empower our users to be more financially independent by gamifying the learning journey through a curated planning feature, suggestions for resources, and other initiatives.
+
+## Tech Stack
+
+Django backend and React frontend.
```

Comment:
Can we break these out into a list? Might make it easier to see everything we have going on as we add to it. Also can we include Python and JS on there? Either as bullets above the frameworks listed or something like Python (Django) Javascript (React). Or may bullet points with the main languages and indented bullet points under each one with the frameworks?

Reply:
Thank you for the feedback - changes have been implemented to list out the stack!